packages/sit-onyx/src/components/OnyxDataGrid/features/sorting/SortAction.ct.tsx

```diff
@@ -0,0 +1,12 @@
+import { test } from "../../../../playwright/a11y";
+import { executeMatrixScreenshotTest } from "../../../../playwright/screenshots";
+import SortAction from "./SortAction.vue";
+
+test.describe("Screenshot tests", () => {
+  executeMatrixScreenshotTest({
+    rows: ["none", "asc", "desc"] as const,
+    columns: ["default"],
```

Comment:
Would prefer to also include hover, active and focus-visible here

Reply:
Seems redundant, as this is just the same as the `OnyxIconButton`